### PR TITLE
Bump dataBrowser version

### PR DESCRIPTION
**Pull request by texasRanger09**

### CODE REVIEW

File: DittoDataBrowser/src/main/java/live/ditto/dittodatabrowser/Documents.kt [hunk 1]
```diff
@@ -97,12 +97,6 @@ fun Documents(collectionName: String, isStandAlone: Boolean) {
                     text = "No Docs",
                     textAlign = TextAlign.Start,
                     color = Color.Blue,
-                    modifier = Modifier
```
Comment (rajramsaroop) [inline DittoDataBrowser/src/main/java/live/ditto/dittodatabrowser/Documents.kt:100]: Why was this removed?

**Review by rajramsaroop — APPROVED** [review]
One question on the code, but everything related to changing the version looks good.

File: DittoDataBrowser/src/main/java/live/ditto/dittodatabrowser/Documents.kt (hunk 1 above)
Comment (texasRanger09) [reply DittoDataBrowser/src/main/java/live/ditto/dittodatabrowser/Documents.kt:100]: It's on the "No Docs" text which means the user cannot click on the text to see the documents. There is nothing to modify when it shows "No Docs".
